daprdocs/content/en/operations/security/mtls.md

```diff
@@ -406,6 +425,8 @@ By default, system services will look for the credentials in `/var/run/dapr/cred
 *Note: If you signed the cert root with a different private key, restart the Dapr instances.*
 
 ## Community call video on certificate rotation
```

Comment:
I realize this video is lost at the end of this article and is best move to the end of the certificate renewal section. Could you move this video up to the end of this section?